### PR TITLE
PLT-2553 Updated backstage page navigation

### PR DESCRIPTION
**Pull request by hmhealey**

### CODE REVIEW

File: webapp/components/backstage/add_outgoing_webhook.jsx [hunk 1]
```diff
@@ -131,16 +131,21 @@ export default class AddOutgoingWebhook extends React.Component {
...
                             />
-                        </h1>
-                    </div>
+                        </Link>
+                        {'>'}
+                        <FormattedMessage
+                            id='add_outgoing_webhook.header'
+                            defaultMessage='Add'
+                        />
+                    </h1>
```
Comment (crspeller) [inline webapp/components/backstage/add_outgoing_webhook.jsx]: This code was just duplicated in the previous files. Move to separate component?

File: webapp/components/backstage/backstage_section.jsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ export default class BackstageSection extends React.Component {
                 <Link
                     className={`${className}-title`}
                     activeClassName={`${className}-title--active`}
-                    onlyActiveOnIndex={true}
+                    onlyActiveOnIndex={false}
```
Comment (crspeller) [inline webapp/components/backstage/backstage_section.jsx]: Default is false so remove?

File: webapp/components/backstage/installed_incoming_webhooks.jsx! [hunk 3]
```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
```
Comment (crspeller) [inline webapp/components/backstage/installed_incoming_webhooks.jsx!]: What's this file about?

File: webapp/components/backstage/add_outgoing_webhook.jsx (hunk 1 above)
Comment (hmhealey) [inline webapp/components/backstage/add_outgoing_webhook.jsx]: K will do

File: webapp/components/backstage/installed_incoming_webhooks.jsx! (hunk 3 above)
Comment (hmhealey) [inline webapp/components/backstage/installed_incoming_webhooks.jsx!]: Oh that was a bad rename. I missed it when I did `git status`

**Comment by hmhealey** [comment]
Looks like mattermod isn't spinning up a new server

Link: http://harrison.spinmint.com:8065/pr2612

Email: test@test.com

Email2: test2@test.com

Password: passwd

**Comment by jwilander** [comment]
+1

**Comment by crspeller** [comment]
+1

**Pull request merged**